helix-view/src/document.rs

```diff
@@ -142,10 +143,15 @@ pub struct Document {
     ///
     /// To know if they're up-to-date, check the `id` field in `DocumentInlayHints`.
     pub(crate) inlay_hints: HashMap<ViewId, DocumentInlayHints>,
+    /// Color swatches for the document
+    ///
+    /// To know if they're up-to-date, check the `id` field in `DocumentColorSwatches`.
+    pub(crate) color_swatches: HashMap<ViewId, DocumentColorSwatches>,
     pub(crate) jump_labels: HashMap<ViewId, Vec<Overlay>>,
     /// Set to `true` when the document is updated, reset to `false` on the next inlay hints
     /// update from the LSP
     pub inlay_hints_oudated: bool,
+    pub color_swatches_outdated: bool,
```

Comment:
Maybe i missed it, but i never see this one set to `true`. The comment above also indicates you should check outdated-ness via IDs. Can we safely remove this property?

Reply:
It's set to `true` on line 1449 of `helix-view/src/document.rs`
I copied this from the inlay hints

Reply:
No worries then, I'm a bit rusty on the codebase so sorry for the stupid comments

Reply:
> No worries then, I'm a bit rusty on the codebase so sorry for the stupid comments

It's completely fine! I appreciate that you're taking the time to review :)